data/mods/Magiclysm/mod_interactions/Defense_Mode/overmap_specials.json

```diff
@@ -61,7 +61,6 @@
     "locations": [ "lake_surface" ],
     "city_distance": [ 40, -1 ],
     "occurrences": [ 100, 100 ],
-    "eoc": { "id": "EOC_DM_TOWER_GENERATE", "condition": { "mod_is_loaded": "defense_mode" } },
```

Comment:
Git seems to have derped at some point and you have a load of unrelated mod changes

Reply:
@Procyonae I don't know, I may have done something weird but indeed I have a bunch of unrelated changes and I don't know why or how to fix it.

The changes are also credited under the username Kwenchin, which I changed some time ago and I'd like this PR to eventually get merged under my current username, if you know how to help.